test/features/login.feature

```diff
@@ -0,0 +1,18 @@
+@smoke
+Feature: Login
+
+    Scenario: Smoke test to ensure Login process works
```

Comment:
Do add tags to these scenarios, like `@login` so we have like groupings if we just need to smoke test a certain section of the system outside of cicd.

Reply:
Have added tags to scenarios.

Reply:
:+1: